### ISSUE TITLE
OPL/Samba: connection dying soon after game boot

### ISSUE THREAD

**Issue by degrenous** [danmons/retronas#131]
OPL samba shares were working great until recently. Often a game will boot but when it reaches a loading/seek point, something stops the samba session and the game locks up because loading's interrupted. Timing isn't consistent either: sometimes it won't even reach boot, or sometimes it will finish loading and work for a few more seconds until it dies again.

samba logs were also unhelpful, showing nothing from what I could see, though the smbd log does show "session closed" a few seconds after "session opened" (I don't know if that's typical behavior). Other samba shares (MiSTer, Windows) continue to work fine. 

I thought it might be disk errors, but it's reproducible with several arbitrarily chosen isos. Playing the same isos over network in emulation (pcsx2) also works fine. Nothing else on the system was messed with so perhaps it's a config file issue? 

Failed fix attempts:
On PS2 side:
-Purged/remade OPL config files on memory card
-Tried both older and newer OPL versions
-Switched from NetBIOS setting to IP
On Linux side:
-Restarted smbd
-Purged & reinstalled samba/smbd
-Even reinstalled Debian completely, to no avail

**Comment by sairuk** [comment]
What version of OPL is this with? 

I believe the only changes we've made in this space was to improve VMC performance.

Yep was under commit https://github.com/danmons/retronas/commit/e6fcdb050114ccc21a58599a1002d3fb912798a1

**Comment by degrenous** [comment]
Originally 1.1.0, then tried with 1.0.0 and a 1.2.0 beta (https://github.com/ps2homebrew/Open-PS2-Loader/releases/tag/latest).

**Comment by sairuk** [comment]
Right thanks for updating can you drop out the contents of your `retronas_ps2.cfg`? So we can make confirm the settings.

`cat /etc/samba/retronas_ps2.conf`

The only change on our side was to add `strict sync = no` and `sync always = yes` to speed up VMC saves

That was on commit e6fcdb050114ccc21a58599a1002d3fb912798a1 does this line up to when you started having issues?

**Comment by degrenous** [comment]
Well, curiously enough, I deleted the ps2 conf file and reran the install to rebuild it, and now games are loading with more success (tried six games and had just one at-boot failure, no during-load failures). Especially curious since as mentioned, I had wiped and started fresh with linux & retronas and still saw the problem, so it would have made the conf from scratch anyway, and I hadn't touched it since. Nevertheless that seems to be the fix.

But to answer your question, it did not line up, as problem was just a few days ago and I was well updated.
